src/javascript/app_2/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -46,20 +47,22 @@ class AccountSwitcher extends React.Component {
         if (!this.props.is_logged_in) return false;
         // TODO: Once we allow other real accounts (apart from CR), assign correct title and group accounts into list with correct account title/types
         // e.g - Real, Financial, Gaming, Investment
+        const real_accounts = this.props.account_list.filter(acc => !acc.is_virtual);
+        const vrt_account   = this.props.account_list.find(acc => acc.is_virtual);
 
-        const main_account_title = localize('Real account');
+        const main_account_title = real_accounts.length > 1 ? localize('Real accounts') : localize('Real account');
 
         return (
             <div className='acc-switcher__list' ref={this.setWrapperRef}>
                 {
                     // Make sure this block is not rendered if there are no real accounts
-                    !!(this.props.account_list.length > 0 && !(this.props.account_list[0].is_virtual)) &&
+                    !!(this.props.account_list.length > 0 && real_accounts.length > 0) &&
```

Comment:
suggestion:
in conditionals better to use `real_accounts.length`  instead of `real_accounts.length > 0`, the first one is shorter and also better in performance

Reply:
Agreed, good idea. Thanks, will make the change in the new branch.